doc/sphinx/addendum/implicit-coercions.rst

```diff
@@ -25,6 +25,13 @@ typed modulo insertion of appropriate coercions. We allow to write:
 Classes
 -------
 
+   .. insertprodn class class
+
+   .. prodn::
+      class ::= Funclass
+      | Sortclass
+      | @smart_global
```

Comment:
I'm surprised to see this at the beginning of the section instead of where the `productionlist` used to be. I wanted to have a look at the rendered HTML to see if I liked the result, but I can't because it's broken.

Reply:
Moved it.  @class and class seem to mean different things here, that could confuse the reader.  Not sure what words to add, though.